assets/generated/api.md

```diff
@@ -11,6 +11,14 @@
 
 {{GetText $m.Operation.Description}}
 
+```sh
+curl -X {{$m.Request.Verb}} \
+    {{$.Swagger.Host}}{{$m.Request.URI}} \
```

Comment:
really nitpicky, but... these are spaces, not tabs :P

Reply:
Indeed vscode uses 4spaces for 1tab. Done!